handsontable/src/shortcuts/__tests__/shortcutManager.spec.js

```diff
@@ -478,4 +478,23 @@ describe('shortcutManager', () => {
 
     document.body.removeChild(externalInputElement);
   });
+
+  it('should check if there is a need of releasing keys on click #32', () => {
```

Comment:
```suggestion
  it('should check if there is a need of releasing keys on click #dev-1025', () => {
```
`#32`? Here shouldn't be [#1025 ](https://github.com/handsontable/dev-handsontable/issues/1025)?